Mathlib/Analysis/Complex/Angle.lean

```diff
@@ -66,4 +69,62 @@ lemma angle_exp_exp (x y : ℝ) :
 lemma angle_exp_one (x : ℝ) : angle (exp (x * I)) 1 = |toIocMod Real.two_pi_pos (-π) x| := by
   simpa using angle_exp_exp x 0
 
+/-!
+### Arc-length and chord-length are equivalent
+
+This section shows that the arc and chord distances between two unit complex numbers are equivalent
+up to a factor of `π / 2`.
+-/
+
+/-- Chord-length is always less than arc-length. -/
+lemma norm_sub_le_angle (hx : ‖x‖ = 1) (hy : ‖y‖ = 1) : ‖x - y‖ ≤ angle x y := by
+  clear a
+  wlog h : y = 1
+  · have := @this (x / y) 1 (by simp only [norm_div, hx, hy, div_one]) norm_one rfl
+    rwa [angle_div_left_eq_angle_mul_right, div_sub_one, norm_div, hy, div_one, one_mul]
+      at this
+    rintro rfl
+    simp at hy
+  subst y
+  rw [norm_eq_abs, abs_eq_one_iff'] at hx
+  obtain ⟨θ, hθ, rfl⟩ := hx
+  rw [angle_exp_one, exp_mul_I, add_sub_right_comm, (toIocMod_eq_self _).2]
+  norm_cast
+  rw [norm_eq_abs, abs_add_mul_I, Real.sqrt_le_left (by positivity), ← _root_.abs_pow, abs_sq]
+  calc
+    _ = 2 * (1 - θ.cos) := by linear_combination θ.cos_sq_add_sin_sq
+    _ ≤ 2 * (1 - (1 - θ ^ 2 / 2)) := by gcongr; exact Real.one_sub_sq_div_two_le_cos
+    _ = _ := by ring
+  · convert hθ
+    ring
+
+/-- Chord-length is always greater than a multiple of arc-length. -/
+lemma mul_angle_le_norm_sub (hx : ‖x‖ = 1) (hy : ‖y‖ = 1) : 2 / π * angle x y ≤ ‖x - y‖ := by
+  clear a
+  wlog h : y = 1
+  · have := @this (x / y) 1 (by simp only [norm_div, hx, hy, div_one]) norm_one rfl
+    rwa [angle_div_left_eq_angle_mul_right, div_sub_one, norm_div, hy, div_one, one_mul]
+      at this
+    rintro rfl
+    simp at hy
+  subst y
+  rw [norm_eq_abs, abs_eq_one_iff'] at hx
+  obtain ⟨θ, hθ, rfl⟩ := hx
+  rw [angle_exp_one, exp_mul_I, add_sub_right_comm, (toIocMod_eq_self _).2]
+  norm_cast
+  rw [norm_eq_abs, abs_add_mul_I]
+  refine Real.le_sqrt_of_sq_le ?_
+  rw [mul_pow, ← _root_.abs_pow, abs_sq]
+  calc
+    _ = 2 * (1 - (1 - 2 / π ^ 2 * θ ^ 2)) := by ring
+    _ ≤ 2 * (1 - θ.cos) := by
+        gcongr; exact Real.cos_quadratic_upper_bound $ abs_le.2 $ Ioc_subset_Icc_self hθ
```

Comment:
I have been told in the past that `<|` is preferred to `$`, so you might change these (and below) to head off that comment.